src/Routes.php

```diff
@@ -133,4 +133,34 @@ public static function plans()
 
         return $anonymous;
     }
+
+    /**
+     * @return \PagarMe\Anonymous
+     */
+    public static function bulkAnticipations()
```

Comment:
Can't we reuse `recipients()->base()`?

Reply:
At first, I thought about reusing the Recipients _base_ or _details_ routes, which would lead to the following:

```php
public function bulkAnticipations()
{
    $anonymous = new Anonymous();

    $anonymous->base = static function ($recipientId) {
        return self::recipients()->details($recipientId) . "/bulk_anticipations";
    }
}
```

IMHO, this would be an improvement with almost no significance. Also, once we did this change, it would make sense to replicate the very same approach for all routes, not only Recipients, which also has no huge impact :wink: